lib/workers/repository/update/branch/index.ts

```diff
@@ -382,6 +386,14 @@ export async function processBranch(
         prNo: branchPr?.number,
         result: 'no-work',
       };
+    }
+    // if the base branch has been changed by user in renovate config, rebase the branch on the new baseBranch
+    // it's fine to use target-branch here because if the target-branch is different ie. changed by user we would stop the processing on line:210
```

Comment:
```suggestion
    // it's fine to use target-branch here because if the target-branch is different ie. changed by user we would stop the processing on line:210
```

lines change all the time - choose another way to reference it